cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -346,20 +348,19 @@ def ssa_multiple_vms(ssa_single_vm, assign_profile_to_vm):
     vms = []
     for item in range(3):
         vm = ssa_single_vm()
-        assign_profile_to_vm(vm)
+        assign_profile_to_vm(vm=vm)
         vms.append(vm)
     return vms
 
 
 @pytest.fixture(scope="module")
 def assign_profile_to_vm(appliance, ssa_policy, request):
     """ Assign policy profile to vm"""
-    def _assign_profile_to_vm(vm):
+    def _assign_profile_to_vm(vm=None):
```

Comment:
Can you clarify why is it needed to assign `vm=None`? How is the policy assigned to `None`?

Reply:
I will remove it.